pom.xml

```diff
@@ -9,7 +9,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.github.gwtbootstrap</groupId>
     <artifactId>gwt-bootstrap</artifactId>
-    <version>2.3.2.2-SNAPSHOT</version>
+    <version>2.3.3-SNAPSHOT</version>
```

Comment:
I wonder if we could just release a 2.3.3. instead of a snapshot.
what do you think?